modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -117,6 +117,8 @@ export interface Output {
   amount: string | number;
   external?: boolean;
```

Comment:
this seems to be redundant with `chain` and `index` being set

elsewhere we split this up into two types

```
export interface BaseOutput {
  address: string;
  amount: string | number;
}

export interface WalletOutput extends Output {
  needsCustomChangeKeySignatureVerification?: boolean;
  chain: number;
  index: number;
}

type Output = BaseOutput | WalletOutput;

function isWalletOutput(o: Output): o is WalletOutput {
  return o.chain !== undefined;
}
```

Reply:
Agreed in principle, but we still need it because we still support legacy format. If it is a PSBT we can explicitly set `external = true` and skip the check. Otherwise if it was just PSBTs I would agree with you